src/manipulation.ts

```diff
@@ -1,3 +1,4 @@
+import * as process from 'process'
```

Comment:
@Dawidpol That's your issue I would say. Do not import process

Reply:
@matiboy i thought of that too, but is not.